code/modules/antagonists/fugitive/fugitive_capture_device.dm

```diff
@@ -0,0 +1,26 @@
+//works similar to the experiment machine (experiment.dm) except it just holds more and more prisoners
+
+/obj/machinery/fugitive_capture
+	name = "bluespace capture machine"
+	desc = "Much, MUCH bigger on the inside to transport prisoners safely."
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "experiment-open"
+	density = TRUE
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF //ha ha no getting out!!
+
+
+/obj/machinery/abductor/experiment/MouseDrop_T(mob/target, mob/user)
+	var/mob/living/L = user
+	if(user.stat || (isliving(user) && (!(L.mobility_flags & MOBILITY_STAND) || !(L.mobility_flags & MOBILITY_UI))) || !Adjacent(user) || !target.Adjacent(user) || !ishuman(target))
+		return
+	var/datum/antagonist/fugitive/fug = L.mind.has_antag_datum(/datum/antagonist/fugitive)
+	if(!fug)
+		to_chat(user, "<span class='warning'>This is not a wanted fugitive!</span>")
+	if(do_after(user, 50, target = src))
+		add_prisoner(target)
+
+/obj/machinery/abductor/experiment/proc/add_prisoner(mob/target, datum/antagonist/fugitive/fug)
+	target.forceMove(src)
+	fug.is_captured = TRUE
```

Comment:
So what happens if someone tries to free them?

Reply:
The machine cannot be broken, only admin fuckery can free then. Might still consider deleting the target and noting if they were alive

Reply:
The machine cannot be broken, only admin fuckery can free then. Might still consider deleting the target and noting if they were alive

Reply:
I kinda wanna make a PR that makes this teleport them to a special centcom jail that they can try to escape from (and get a minor greentext or smthn) for funsies

Reply:
@MrDoomBringer please do that after this pr is merged, i'm also gonna follow the pr up with rugged bounty hunters using boomsticks, hooks and flamethrowers